external_equipment/sam_joy_controllers/sam_joy_controllers/sam_joy_xbox/package.xml

```diff
@@ -12,10 +12,7 @@
   <test_depend>ament_pep257</test_depend>
   <test_depend>python3-pytest</test_depend>
   <depend>rclpy</depend>
-  <depend>sensor_msgs</depend>
```

Comment:
removed a few too many things :D 
keep std_msgs and sensor_msgs, these are installable with rosinstall since they are distributed through apt and such.